users/userRouter.js

```diff
@@ -1,47 +1,149 @@
-const express = 'express';
+const express = require("express");
 
-const router = express.Router();
-
-router.post('/', (req, res) => {
+const userDb = require("./userDb");
 
-});
+const postDb = require("../posts/postDb");
 
-router.post('/:id/posts', (req, res) => {
+const router = express.Router();
 
+router.post("/", validateUser, async (req, res) => {
+  const addUser = await userDb.insert(req.body);
+  if (addUser) {
+    res.status(201).json(addUser);
+  } else {
+    res
+      .status(500)
+      .json({ message: "Something went wrong, should not be getting here" });
+  }
```

Comment:
This is fine, it should be noted that if you chose to do so you could utilize trycatch to find out specifically what error you were getting.